slick/src/main/scala/slick/ast/Type.scala

```diff
@@ -48,7 +48,7 @@ trait AtomicType extends Type {
 }
 
 final case class StructType(elements: ConstArray[(TermSymbol, Type)]) extends Type {
-  override def toString = "{" + elements.iterator.map{ case (s, t) => s + ": " + t }.mkString(", ") + "}"
+  override def toString = "{" + elements.iterator.map{ case (s, t) => s.toString + ": " + t }.mkString(", ") + "}"
```

Comment:
You can also use an interpolator here. I believe it does slightly better optimization for the string concatenation